packages/FormElement/README.md

```diff
@@ -140,19 +131,20 @@ Using with DOM input
 ```jsx
 import FormElement from "@paprika/form-element";
 
-<FormElement hasRequiredLabel={hasRequiredLabel} label="Name">
+<FormElement hasRequiredLabel={hasRequiredLabel}>
+  <Label>Name</Label>
   <FormElement.Content>
-    {({ idForLabel, ariaDescribedBy }) => (
+    {a11yProps => (
       <input
         aria-required={hasRequiredLabel}
-        aria-describedby={ariaDescribedBy}
+        aria-describedby={a11yProps[aria-describedby]}
         aria-invalid={Boolean(errorText.length)}
         disabled={isDisabled}
-        id={idForLabel}
+        id={a11yProps.id}
```

Comment:
Can you not spread them on a native `<input>`?

Reply:
Yes but then we get warnings about `refLabel` not being an supported attribute of an input element. Which is perhaps fine 🤷‍♂️  But i would have thought we would want to avoid. I would think most of the time developers would be using the paprika `Input` element instead which deletes the `refLabel` prop for them...

Reply:
I see 🤔 

Well, I think the README should be focused exclusively on the most common use cases, we can use the Docs page or the Examples stories to demonstrate these edgier cases.  Maybe just explain this in the `NativeInput` story, and cut this example from README.